zk_shell/shell.py

```diff
@@ -2778,6 +2778,35 @@ def complete_reconfig(self, cmd_param_text, full_cmd, *rest):
         completers = [complete_cmd, complete_arg, complete_config]
         return complete(completers, cmd_param_text, full_cmd, *rest)
 
+    @ensure_params(Required("fmtstr"), MultiOptional("cmds"))
+    def do_echo(self, params):
+        """
+\x1b[1mNAME\x1b[0m
```

Comment:
Here's what I was thinking, but up to you with whatever looks better.

Something like https://github.com/tartley/colorama#colored-output:

```
"""
%sNAME%s
A cool name.
""" % (Fore.RED, Style.RESET_ALL)